test/functional/vimscript/exepath_spec.lua

```diff
@@ -32,11 +29,32 @@ describe('exepath()', function()
   end)
 
   if is_os('win') then
+    it('returns 1 for commands in $PATH (Windows)', function()
+      local exe = 'ping'
+      matches(exe .. '%.EXE$', call('exepath', exe))
+    end)
+
     it('append extension if omitted', function()
       local filename = 'cmd'
       local pathext = '.exe'
       clear({env={PATHEXT=pathext}})
       eq(call('exepath', filename..pathext), call('exepath', filename))
     end)
+
+    it('returns file WITH extension if files both with and without extension exist in $PATH', function()
+      local ext_pat = '%.CMD$'
+      find_dummies(ext_pat)
+      set_shell_powershell()
```

Comment:
this is failing because `find_dummies` changed `$PATH` and `powershell` isn't found in the new `$PATH`

```
test\functional\helpers.lua:557: assertion failed!
stack traceback:
	test\functional\helpers.lua:557: in function 'set_shell_powershell'
	test/functional\vimscript\exepath_spec.lua:47: in function <test/functional\vimscript\exepath_spec.lua:44>
```

Reply:
I ended up using `eval('$PATH')`. This works now on Windows. And `find_dummies` changes the PATH back to the previous one.